app/code/Stanislavz/Chat/view/frontend/web/css/source/_module.less

```diff
@@ -0,0 +1,151 @@
+.chat-clearfix() {
+  &:after {
+    clear: both;
+  }
+}
+
+.hidden {
```

Comment:
@ahrrhy 
1) This does not look like a mixin. And must not appear directly in the file
2) Do not use classes to hide elements. Use jQuery fadeIn() and fadeOut() or similar things. Use `visibility` binding in Knockout.